Content.Server/Shuttles/Components/ShuttleConsoleComponent.cs

```diff
@@ -14,5 +14,11 @@ public sealed partial class ShuttleConsoleComponent : SharedShuttleConsoleCompon
         /// </summary>
         [DataField("zoom")]
         public Vector2 Zoom = new(1.5f, 1.5f);
+
+        /// <summary>
+        /// Should this console have access to restricted FTL destinations?
+        /// </summary>
+        [ViewVariables(VVAccess.ReadWrite), DataField("whitelistSpecific")]
+        public List<EntityUid> FTLWhitelist = new List<EntityUid>();
     }
```

Comment:
The problem with how this is done is you need to continuously cache this and it's guaranteed to serialize invalid data when map saving. It's far better to just check this at runtime.

Reply:
Just to clarify; instead of having the CD (and any other FTL whitelist method) add the whitelisted destinations to a list stored by the console, the check for whether a CD is inserted and what its coordinates are should be done in CanFTLTo in SharedShuttleSystem?

Reply:
Yeah, so just pass in the console to some shared method (I think CanFTLTo exists but it's slightly different) and that way you don't need to do any tracking.